### PR TITLE
BREAKING CHANGE: Rename ScrollView `indicators` to `overflowDecorators` #DS-825

### PR DESCRIPTION
**Pull request by crishpeen**
Please see also commit messages. I tried to introduce some BC commit message template with a migration guide.

### CODE REVIEW

**Review by literat — APPROVED** [review]

**Review by tomassychra — APPROVED** [review]

File: packages/web-react/src/components/ScrollView/README.md [hunk 1]
```diff
@@ -40,21 +40,21 @@ limit using inline styles for demonstration purposes only.
 </ScrollView>
 ```
 
-## Scrolling Indicators
+## Scrolling Overflow Decorators
```
Comment (adamkudrna) [inline packages/web-react/src/components/ScrollView/README.md:43]: Maybe just:

```suggestion
## Overflow Decorators
```

File: packages/web-react/src/components/ScrollView/README.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ Or both:
 | --------------------- | ---------------------------- | ---------- | -------- | ---------------------------------- |
 | `children`            | `ReactNode`                  | -          | yes      | ScrollView children's nodes        |
 | `direction`           | `horizontal`, `vertical`     | `vertical` | no       | Direction of the wrapper           |
-| `edgeIndicators`      | `borders`, `shadows`, `both` | `shadows`  | no       | ScrollView edge indicators         |
+| `overflowDecorators`  | `borders`, `shadows`, `both` | `shadows`  | no       | ScrollView overflow decorators     |
```
Comment (adamkudrna) [inline packages/web-react/src/components/ScrollView/README.md:72]: 🔤

File: packages/web-react/src/components/ScrollView/ScrollView.tsx [hunk 3]
```diff
@@ -4,13 +4,13 @@ import { SpiritScrollViewProps } from '../../types';
...
 import { useScrollPosition } from './useScrollPosition';
-import { SCROLL_VIEW_DEFAULT_EDGE_INDICATOR, SCROLL_VIEW_DEFAULT_DIRECTION } from './constants';
+import { SCROLL_VIEW_DEFAULT_DIRECTION, SCROLL_VIEW_DEFAULT_OVERFLOW_DECORATOR } from './constants';
 
 const ScrollView = (props: SpiritScrollViewProps) => {
   const {
     children,
     direction = SCROLL_VIEW_DEFAULT_DIRECTION,
-    edgeIndicators = SCROLL_VIEW_DEFAULT_EDGE_INDICATOR,
+    overflowDecorators = SCROLL_VIEW_DEFAULT_OVERFLOW_DECORATOR,
```
Comment (adamkudrna) [inline packages/web-react/src/components/ScrollView/ScrollView.tsx:13]: 🔤 

Here and everywhere else…

File: packages/web-twig/src/Resources/components/ScrollView/ScrollView.stories.twig [hunk 4]
```diff
@@ -11,8 +11,8 @@
     {# ScrollView Horizontal with Container Breakout #}
     {% include '@components/ScrollView/stories/ScrollViewHorizontalBreakout.twig' %}
 
-    {# ScrollView Scrolling Indicators #}
-    {% include '@components/ScrollView/stories/ScrollViewScrollingIndicators.twig' %}
+    {# ScrollView Scrolling Overflow Decorators #}
```
Comment (adamkudrna) [inline packages/web-twig/src/Resources/components/ScrollView/ScrollView.stories.twig:14]:
```suggestion
    {# ScrollView Overflow Decorators #}
```

File: packages/web-twig/src/Resources/components/ScrollView/ScrollView.stories.twig [hunk 5]
```diff
@@ -11,8 +11,8 @@
...
+    {% include '@components/ScrollView/stories/ScrollViewScrollingOverflowDecorators.twig' %}
```
Comment (adamkudrna) [inline packages/web-twig/src/Resources/components/ScrollView/ScrollView.stories.twig:15]:
```suggestion
    {% include '@components/ScrollView/stories/ScrollViewOverflowDecorators.twig' %}
```

File: packages/web-twig/src/Resources/components/ScrollView/stories/ScrollViewScrollingOverflowDecorators.twig [hunk 6]
```diff
@@ -1,13 +1,13 @@
 <section class="docs-Section">
 
-    <h2 class="docs-Heading">Scrolling Indicators</h2>
+    <h2 class="docs-Heading">Scrolling Overflow Decorators</h2>
```
Comment (adamkudrna) [inline packages/web-twig/src/Resources/components/ScrollView/stories/ScrollViewScrollingOverflowDecorators.twig:3]:
```suggestion
    <h2 class="docs-Heading">Overflow Decorators</h2>
```

Please shorten it also in the other files. ⬇️

**Comment by adamkudrna** [comment]
> Please refer back to this commit message or reach out to our team if you encounter any issues during migration.

Maybe "guide" instead of "commit message"? Because the guide will be rendered also in other contexts like MD.

**Review by adamkudrna — COMMENTED** [review]
Cool, the guides look 👍🏻.

File: packages/web-react/src/components/ScrollView/README.md (hunk 2 above)
Comment (literat) [reply packages/web-react/src/components/ScrollView/README.md]: Fixed.

File: packages/web-react/src/components/ScrollView/ScrollView.tsx (hunk 3 above)
Comment (literat) [reply packages/web-react/src/components/ScrollView/ScrollView.tsx]: Fixed.

File: packages/web-twig/src/Resources/components/ScrollView/stories/ScrollViewScrollingOverflowDecorators.twig (hunk 6 above)
Comment (literat) [reply packages/web-twig/src/Resources/components/ScrollView/stories/ScrollViewScrollingOverflowDecorators.twig]: Fixed.

File: packages/web-twig/src/Resources/components/ScrollView/ScrollView.stories.twig (hunk 5 above)
Comment (literat) [reply packages/web-twig/src/Resources/components/ScrollView/ScrollView.stories.twig]: Fixed.

File: packages/web-twig/src/Resources/components/ScrollView/ScrollView.stories.twig (hunk 4 above)
Comment (literat) [reply packages/web-twig/src/Resources/components/ScrollView/ScrollView.stories.twig]: Fixed.

File: packages/web-react/src/components/ScrollView/README.md (hunk 1 above)
Comment (literat) [reply packages/web-react/src/components/ScrollView/README.md]: Fixed.

**Review by adamkudrna — APPROVED** [review]

**Pull request merged**